zerver/lib/test_runner.py

```diff
@@ -218,11 +225,9 @@ def _replacement_destroy_test_db(self: DatabaseCreation,
                        % (self.connection.ops.quote_name(test_database_name),))
 DatabaseCreation._destroy_test_db = _replacement_destroy_test_db
 
-def destroy_test_databases(database_id: Optional[int]=None) -> None:
-    """
-    When database_id is None, the name of the databases is picked up
-    by the database settings.
-    """
+def destroy_test_databases(worker_id: int) -> None:
+    """Modified from the Django original to """
+    database_id = random_id_range_start + worker_id
```

Comment:
(Incomplete comment?)

It looks like we are [allowed to use a string](https://code.djangoproject.com/ticket/28578) rather than a number here, so that we don’t need to rely on assumptions like the number of workers being less than 100.

Reply:
That's only in Django 2.0+.

Reply:
Added a note to https://github.com/zulip/zulip/issues/11341